templates/CRM/Contact/Form/Edit/CustomData.tpl

```diff
@@ -10,8 +10,6 @@
 
 {foreach from=$groupTree item=cd_edit key=group_id}
   {if $cd_edit.is_multiple eq 1}
-    {assign var=tableID value=$cd_edit.table_id}
-    {assign var=divName value=$group_id|cat:"_$tableID"}
```

Comment:
divName is never used - not even before https://github.com/civicrm/civicrm-core/commit/3c07d6c9416967c2479e11a2e81d995c7b8bec96#diff-6ce562a05df7bee3378066d0226a751e359f990cadbf0afc2999c61682c6d39fL16 & tableID is only set her to set divName